.circleci/config.yml

```diff
@@ -128,6 +142,18 @@ jobs:
           name: Auto-rebase
           command: bash .circleci/scripts/auto-rebase.sh
 
+  rebase-and-unit-test:
```

Comment:
confirming that we are testing only with one PHP version for merchant branches and only unit tests?

Reply:
This workflow should be triggered 3 times per PR with each php version. Running rebase and unit tests

Reply:
^ is not the case right now? also if you need to trigger it 3 times you need to pass phpo version in the name of the job otherwise circle reporting will be off